backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/init/CassandraTypeProviderTest.java

```diff
@@ -58,13 +63,13 @@ public void setUp() {
                         .addColumn(PROPERTY, text()))));
             }
         };
-        cassandra = CassandraCluster.create(module);
+        cassandra = CassandraCluster.create(module, cassandraServer.getIp(), cassandraServer.getBindingPort());
```

Comment:
IMO you should have a : 

```
cassandraServer.createCluster(module);
```

That seems easier for a test writer...

Reply:
DockerCassandraRule is intended as a very reusable component. I will probably contribute it to testcontainers soon. I don't think mixing it with the driver part is a good idea as it mixes responsibility.

Reply:
:+1: